crds/jwst/specs/fgs_pars-darkpipeline.rmap

```diff
@@ -0,0 +1,23 @@
+header = {
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'Hand-made, SCSB, CRDS-416, 2020-10',
+    'file_ext' : '.asdf',
+    'filekind' : 'pars-darkpipeline',
+    'filetype' : 'pars-darkpipeline',
+    'instrument' : 'FGS',
+    'mapping' : 'REFERENCE',
+    'name' : '',
+    'observatory' : 'JWST',
+    'parkey' : ((), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
```

Comment:
Do we need the first component of the key if it's empty?  I mean, is it possible to do this:

```
'parkey' : (('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME'),),
```
then:
```
selector = UseAfter({
})
```

Reply:
probably not. just copied from some existing one that did the same thing. however, i like it because if a selector becomes necessary, the placeholder is there, for those of us who don't remember ****.